usr/tgtd.c

```diff
@@ -894,6 +894,8 @@ static int target_delete(const _ha_request *reqp,
 	int rc  = 0;
 	int len = 0;
 	int force = -1;
```

Comment:
Why initial value is -1 and not simply 1. If possible, make it a bool variable.

Reply:
Done